lib-es5/v2/search.js

```diff
@@ -111,6 +138,37 @@ var Search = function () {
       options = options || {};
       return api.search(this.to_query(), options, callback);
     }
+  }, {
+    key: 'to_url',
+    value: function to_url(ttl, next_cursor) {
+      var options = arguments.length > 2 && arguments[2] !== undefined ? arguments[2] : {};
+
+      var apiSecret = 'api_secret' in options ? options.api_secret : config().api_secret;
+      if (!apiSecret) {
+        throw new Error('Must supply api_secret');
+      }
+
+      var signingAlgorithm = options.signature_algorithm || config().signature_algorithm || DEFAULT_SIGNATURE_ALGORITHM;
+
+      var urlTtl = ttl || this._ttl;
+
+      var query = this.to_query();
+
+      var urlCursor = next_cursor;
+      if (query.next_cursor && !next_cursor) {
+        urlCursor = query.next_cursor;
+        delete query.next_cursor;
+      }
+
+      var data = clear_blank(query);
+      var encodedQuery = base64Encode(JSON.stringify(data));
+
+      var urlPrefix = build_distribution_domain(options);
+
+      var signature = compute_hash(`${urlTtl}${encodedQuery}${apiSecret}`, signingAlgorithm, 'hex');
+
+      return urlCursor ? `${urlPrefix}/search/${signature}/${urlTtl}/${encodedQuery}/${urlCursor}` : `${urlPrefix}/search/${signature}/${urlTtl}/${encodedQuery}/`;
```

Comment:
Please rewrite the code not to duplicate `${urlPrefix}/search/${signature}/${urlTtl}/${encodedQuery}` part

Reply:
Will do.